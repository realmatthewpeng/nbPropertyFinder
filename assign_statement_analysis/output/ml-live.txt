Total number of lines found: 7

df_valid = shuffled.iloc[trainsize:(trainsize+validsize), :]
df_train = shuffled.iloc[:trainsize, :]
df_valid = pd.read_csv('data/taxi-valid.csv', header=None, names=columns)
df_train = pd.read_csv('data/taxi-train.csv', header=None, names=columns)
df_valid = pd.read_csv('./data/taxi-valid.csv', header = None, names = CSV_COLUMNS)
df_train = pd.read_csv('./data/taxi-train.csv', header = None, names = CSV_COLUMNS)
df = pd.DataFrame({'Lab' : pd.Series(['1a', '2-3', '4a', '4b', '4c']),'Method' : pd.Series(['Heuristic Benchmark', 'tf.learn', '+Feature Eng.', '+ Hyperparam', '+ 2m rows']),'RMSE': pd.Series([8.026, 9.4, 8.3, 5.0, 3.03]) })

Incorrect Lines:


Precision = 1.0
Recall = 0.7777777777777778

tollrides = trips[trips['tolls_amount'] > 0]
trips = trips_in.copy(deep=True)
