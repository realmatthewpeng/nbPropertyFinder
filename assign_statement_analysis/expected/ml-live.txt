tollrides = trips[trips['tolls_amount'] > 0]
trips = trips_in.copy(deep=True)
df_train = shuffled.iloc[:trainsize, :]
df_valid = shuffled.iloc[trainsize:(trainsize+validsize), :]
rate = df_train['fare_amount'].mean() / estimate_distance(df_train).mean()
df_train = pd.read_csv('data/taxi-train.csv', header=None, names=columns)
df_valid = pd.read_csv('data/taxi-valid.csv', header=None, names=columns)
df_train = pd.read_csv('./data/taxi-train.csv', header = None, names = CSV_COLUMNS)
df_valid = pd.read_csv('./data/taxi-valid.csv', header = None, names = CSV_COLUMNS)
df = pd.DataFrame({'Lab' : pd.Series(['1a', '2-3', '4a', '4b', '4c']),'Method' : pd.Series(['Heuristic Benchmark', 'tf.learn', '+Feature Eng.', '+ Hyperparam', '+ 2m rows']),'RMSE': pd.Series([8.026, 9.4, 8.3, 5.0, 3.03]) })